.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -3,3 +3,6 @@ contact_links:
   - name: Account Limit Increase Request
     url: https://mailxto.com/xy5ijb
     about: Submit a request to increase the size limit on your Web3.Storage account!
+  - name: Pinning Service Access Request
+    url: https://mailxto.com/h58kmo
+    about: Submit a request to access Web3.Storage's IPFS pinning service in your account!
```

Comment:
needs a new line?

Reply:
@alexandrastoica Functionally, it doesn't; unless you mean file linting? 😬

Reply:
Just linting, saw it flagged in the diff